no-idea.md

```diff
@@ -0,0 +1,18 @@
+- Patient
+  - extension ?
+  - NRIC ?
+  
+- Specimen
+  - coding is correct ? http://snomed.info/id/258500001 (looks ok based on example)
+
+
+https://confluence.ihtsdotools.org/download/attachments/106701475/doc_Covid19Guide_Current-en-US_INT_20200423.pdf?api=v2
+=> specimen 258500001
+=> rt pcr -> 444077007
+=> serology -> 88667002
```

Comment:
ive checked with MOH that the pcr and serology code is correct

Reply:
which one loinc or snomed ? and why ?

Reply:
ionic. i didnt ask why. i thought it was a good thing haha